projects/go-lib/src/lib/components/go-table/go-table.component.ts

```diff
@@ -409,14 +414,19 @@ export class GoTableComponent implements OnInit, OnChanges, AfterViewInit {
   }
 
   private setupSearch(): void {
+    if (this.localTableConfig.searchConfig.searchTerm) {
+      this.searchTerm.setValue(this.localTableConfig.searchConfig.searchTerm);
```

Comment:
This won't actually filter on the table because the `valueChanges` subscription hasn't been set up yet right? Should this be moved after the subscription has been set up?

Reply:
This is correct, I think I've fixed the issue. I had to add it to the `ngAfterViewInit` method because `performSearch` uses the columns, which are only present `afterViewInit` so technically this will render the table, then search.

Not sure if there is a great way around this without changing how we search client side.